app/views/decidim/vocdoni/admin/answers/new.html.erb

```diff
@@ -1,4 +1,5 @@
-<% add_decidim_page_title(t(".title")) %>
```

Comment:
On this case and in almost all the pages that I've reviewed, as we're removing this helper we're losing the HTML page titles. Can you add it again on this and all the pages please?